src/core/ledger/grainAllocation.js

```diff
@@ -66,17 +66,20 @@ export type AllocationPolicy =
 export type BalancedPolicy = {|
   +policyType: Balanced,
   +budget: G.Grain,
+  +credMapping?: CredMapping,
 |};
 
 export type ImmediatePolicy = {|
   +policyType: Immediate,
   +budget: G.Grain,
+  +credMapping?: CredMapping,
 |};
 
 export type RecentPolicy = {|
   +policyType: Recent,
   +budget: G.Grain,
   +discount: Discount,
+  +credMapping?: CredMapping,
```

Comment:
This is a no-go. The Policies need to be serializable (they are included in the ledger event), but functions are not serializable.

Reply:
Ok, you had mentioned this previously but I had forgotten.  I'll close this PR.